aws/buckets/athena.tf

```
@@ -0,0 +1,22 @@
+#
+# Holds Athena query resuts
+#
+module "athena_bucket" {
+  source            = "github.com/cds-snc/terraform-modules//S3?ref=17994187b8628dc5decf74ead84768501378df4c" # ref for v10.0.0
+  bucket_name       = "cds-data-lake-athena-${var.env}"
```

Unfortunately this will clash with the S3 bucket name that's in the `DataLake-Production` AWS account once this moves to prod.   Could you add a `forms` segment in the name perhaps?
```suggestion
  bucket_name       = "cds-forms-data-lake-athena-${var.env}"
```